assets/airports/lipz.json

```diff
@@ -0,0 +1,2329 @@
+            ]
+        },
+        "VIC7V": {
+            "icao": "VIC7V",
+            "name": "Vicenza Seven Victor",
+            "rwy": {
+                "LIPZ04R": []
+            },
+            "body": [["^_PZ402", "A10+|S230-"],"_PZ451","_PZ452",["_PZ453", "A50+"],["VIC",  "A120+"]],
+            "exitPoints": {
+                "VIC": []
+            },
+            "draw": [
+                ["_PZ402", "_PZ451", "_PZ452", "_PZ453", "VIC"]
+            ]
+        },
+        "VIC7W": {
+            "icao": "VIC7W",
+            "name": "Vicenza Seven Whiskey",
+            "rwy": {
+                "LIPZ04R": []
+            },
+            "body": [["^_PZ402", "A10+|S230-"],"_PZ431","_PZ432",["_PZ454", "A50+"],"_PZ434","_PZ437",["_PZ438", "A105+"],["_PZ439", "A140+"],"VIC"],
+            "exitPoints": {
+                "VIC": []
+            },
+            "draw": [
+                ["_PZ402", "_PZ431", "_PZ432", "_PZ454", "_PZ434", "_PZ437", "_PZ438", "_PZ439", "VIC"]
+            ]
+        },
+        "VIC7X": {
+            "icao": "VIC7X",
+            "name": "Vicenza Seven X-Ray",
+            "rwy": {
+                "LIPZ04R": []
+            },
+            "body": [["_PZ488", "A07+|S200-"],["_PZ489", "A34+|S200-"],["_PZ490", "A40+|S200-"],["_PZ481", "A100+"],["VIC", "A120+"]],
+            "exitPoints": {
+                "VIC": []
+            },
+            "draw": [
+                ["_PZ488", "_PZ489", "_PZ490", "_PZ481", "VIC"]
+            ]
+        },    
+        "VIC9E": {
+            "icao": "VIC9E",
+            "name": "Vicenza Niner Echo",
+            "rwy": {
+                "LIPZ22L": []
+            },
+            "body": [["LAREN", "A90+"],["_PZ468", "A100+"],["VIC", "A120+"]],
+            "exitPoints": {
+                "VIC": []
+            },
+            "draw": [
+                ["LAREN", "_PZ468", "VIC"]
+            ]
+        },
+        "VIC9Y": {
+            "icao": "VIC9Y",
+            "name": "Vicenza Niner Yankee",
+            "rwy": {
+                "LIPZ04R": []
+            },
+            "body": [["^_PZ402", "A10+|S230-"],"_PZ431",["_PZ432", "A40+"],"_PZ416",["_PZ455", "A60+"],["VIC",  "A120+"]],
+            "exitPoints": {
+                "VIC": []
+            },
+            "draw": [
+                ["_PZ402", "_PZ431", "_PZ432", "_PZ416", "_PZ455", "VIC"]
+            ]
+        }
+    },
+    "stars": {
+        "ALBET1E": {
+            "icao": "ALBET1E",
+            "name": "Albet One Echo",
+            "entryPoints": {
+                "ALBET": [
+                    ["ALBET", "A100+|A130-|S270-"]
+                ]
+            },
+            "body": [["PZ768", "A90+"],["PZ760", "A90+"],["PZ755", "A90-|A60+|S230-"],["PZ750", "A90-|A60+|S230-"],["PZ745", "A90-|A60+|S230-"],["PZ740", "A90-|A60+|S210-"],["PZ735", "A50+|S210-"],["PZ730", "A50+|S210-"],["PZ725", "A40+|S210-"],["LAREN", "A40+|S210"]],
```

Comment:
The restrictions here are not the same as I see on my charts. Restrictions should be defined _exactly_ as in the charts (with a few very rare exceptions). It is possible we're using different charts that show different restrictions but it's unlikely.

Additionally, when implementing fixes with a lower and an upper restriction the lower should be first, then the upper. e.g. `["ALBET", "A100+|A130-|S270-"]`

Reply:
![image](https://user-images.githubusercontent.com/43710005/74238784-c92b6c80-4cd6-11ea-858e-1d5bae3d9d6c.png)
Italian AIP from https://www.enav.it/enavWebPortalStatic/AIP/AIP/ad/ad2/ADPPZ4-1.pdf

Reply:
Very good, disregard then 👍